pkg/glib/glib.go

```diff
@@ -5,36 +5,61 @@ import (
 	"context"
 	"fmt"
 	"io/ioutil"
-	"log"
 
 	"github.com/kubermatic-labs/gman/pkg/config"
 	password "github.com/sethvargo/go-password/password"
 	"golang.org/x/oauth2/google"
 	admin "google.golang.org/api/admin/directory/v1"
+	groupssettings "google.golang.org/api/groupssettings/v1"
 	"google.golang.org/api/option"
 )
 
-// CreateDirectoryService() creates a client for communicating with Google APIs,
-// returns an Admin SDK Directory service object authorized with.
+// NewDirectoryService() creates a client for communicating with Google Directory API,
+// returns a service object authorized to perform actions in Gsuite.
 func NewDirectoryService(clientSecretFile string, impersonatedUserEmail string) (*admin.Service, error) {
 	ctx := context.Background()
 
 	jsonCredentials, err := ioutil.ReadFile(clientSecretFile)
 	if err != nil {
-		return nil, fmt.Errorf("ReadFile(clientSecretFile): %v", err)
+		return nil, fmt.Errorf("Unable to read json credentials (clientSecretFile): %v", err)
 	}
 
-	config, err := google.JWTConfigFromJSON(jsonCredentials, admin.AdminDirectoryUserScope, admin.AdminDirectoryGroupScope, admin.AdminDirectoryGroupMemberScope, admin.AdminDirectoryOrgunitScope)
+	config, err := google.JWTConfigFromJSON(jsonCredentials, admin.AdminDirectoryUserScope, admin.AdminDirectoryGroupScope, admin.AdminDirectoryGroupMemberScope, admin.AdminDirectoryOrgunitScope, admin.AdminDirectoryResourceCalendarScope)
 	if err != nil {
-		return nil, fmt.Errorf("JWTConfigFromJSON: %v", err)
+		return nil, fmt.Errorf("Unable to process credentials: %v", err)
```

Comment:
error strings should always start with a lowercase letter, because errors are often concatenated like "something failed: this did not work: omg what?: file does not exist`. Exception is of course if the first word is a special noun or initials (like `LoadBalancer could not be reconciled`).

Reply:
See https://github.com/golang/go/wiki/CodeReviewComments#error-strings for more information.